code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -120,8 +120,8 @@
 			new /obj/item/card/emag(src) // 6 tc
 
 /obj/item/storage/box/syndicate/bundle_B/PopulateContents()
-	switch (pickweight(list( "bond" = 2, "neo=1", "ninja" = 1, "darklord" = 1, "white_whale_holy_grail" = 2, "mad_scientist" = 2, "bee" = 2, "mr_freeze" = 2)))
-		if("bond")
+	switch (pickweight(list( "bond" = 2,"neo=1", "brother" = 1, "ninja" = 1, "darklord" = 1, "white_whale_holy_grail" = 2, "mad_scientist" = 2, "bee" = 2, "mr_freeze" = 2)))
+    	if("bond")
```

Comment:
There's something up with the indentation here which is what is throwing travis off to start with.

Reply:
Still not right

Reply:
Try copying the the three lines from /obj to if, from the original code and pasting it in again (adding the brother)?

Reply:
Also just noticed neo is wrong, should say "neo" = 1 but it says "neo=1"